frontend/views/profile.html

```diff
@@ -0,0 +1,95 @@
+<div class="me" ng-hide="!user">
+  <div ng-hide="edit" class="pull-right btn-group" role="group">
+    <button class="btn btn-default" type="submit" ng-click="editProfile()">Update</button>
+    <button class="btn btn-warning" type="submit" ng-click="logout()">Logout</button>
+  </div>
+  <div class="page-header">
+    <h2>
+      {{user.username}}
+    </h2>
+  </div>
+  <div ng-hide="edit" class="row">
+    <div class="col-sm-10">
+      <dl class="dl-horizontal">
+        <dt>Username</dt><dd>{{user.username}}</dd>
+        <dt>Email</dt><dd>{{user.email}}</dd>
+
+        <dt ng-hide="!user.catchphrase">Catchphrase</dt>
+        <dd ng-hide="!user.catchphrase">“{{user.catchphrase}}”</dd>
+
+        <dt ng-show="user.roles.length">Roles</dt>
+        <dd ng-show="user.roles.length">
+          <ul class="list-unstyled">
+            <li ng-repeat="role in user.roles">{{role}}</li>
+          </ul>
+        </dd>
+      </dl>
+    </div>
+    <div class="col-sm-2">
+      <style>
```

Comment:
Yeah, I hated doing this, but I wasn't certain if there was a more angular way to do it. That's why I inlined it, because I knew it would bother you ;-)